.github/workflows/release.yml

```diff
@@ -481,6 +481,10 @@ jobs:
           go build
           go test -v ./tests/
         working-directory: go
```

Comment:
Should this step have a conditional `if: runner.os != 'Linux'`?

Reply:
I don't think so, why? We wanna run the tests on all platforms.

Reply:
Ahhhh, I was misreading what's going on here. So we've changed the _build_ step to use the musl lib instead of the glibc lib, and we're adding Alpine to the set of test environments to make sure that the new musl-powered lib works on macOS, Windows, Ubuntu, _and_ a musl-based distro.